src-frontend/app/routes/index.js

```diff
@@ -0,0 +1,4 @@
+import Application from '../routes/application';
```

Comment:
Can we do `src-frontend/routes/application` instead? In all of the files :innocent: